Mathlib/SetTheory/Ordinal/Nimber.lean

```diff
@@ -29,6 +30,10 @@ To reduce API duplication, we opt not to implement operations on `Nimber` on `Or
 isomorphisms `Ordinal.toNimber` and `Nimber.toOrdinal` allow us to cast between them whenever
 needed.
 
+We also define notation `∗o` for `Ordinal.toNimber o` in the `Nimber` namespace. Note that for
```

Comment:
```suggestion
## Notations

We also define notation `∗o` for `Ordinal.toNimber o` in the `Nimber` namespace. Note that for
```
and the notation section should go above the implementation notes section (https://leanprover-community.github.io/contribute/doc.html).